pyccel/codegen/printing/ccode.py

```diff
@@ -568,6 +568,8 @@ def _print_FuncAddressDeclare(self, expr):
         return '{}(*{})({});'.format(ret_type, name, arg_code)
 
     def _print_Declare(self, expr):
+        if isinstance(expr.variable.dtype, NativeInteger) :
+            self._additional_imports.add('stdint')
```

Comment:
This is definitely not the right place as it will be missed if the argument is an `int`. Maybe in `find_in_dtype_registry`. But your original location may be correct. That is why ideally I would like Noureddine's opinion

Reply:
it was nourddine who suggested that I print include  only when it needed

Reply:
Ah ok :+1:

Reply:
I still think this is causing the windows failure though. If you translate the following function. Is stdint included?
```
@types('int')
def f(a):
    print(a)
```